ci-operator/step-registry/image-based-install-operator/conf/devscripts/image-based-install-operator-conf-devscripts-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: image-based-install-operator-conf-devscripts
+  from: dev-scripts
+  commands: image-based-install-operator-conf-devscripts-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 500Mi
+  env:
+    - name: IP_STACK
+      default: v4
+      documentation: |-
+        Configures the IP_STACK for the devscripts cluster. The possible values are v4, v6 or
+        v4v6 for dual stack. The default is v4. For more information on these values refer to
+        https://github.com/openshift-metal3/dev-scripts/blob/master/config_example.sh
+    - name: NUM_EXTRA_WORKERS
+      default: "1"
+      documentation: Indicate number of extra VMs to create but not deploy.
+    - name: GO_VERSION
+      default: "1.22.7"
```

Comment:
Do we have to pin a go version ?

Reply:
yes, when we run `make deploy` we need a high enough go version for controller-gen